coops-ui/tests/unit/Directors.spec.ts

```diff
@@ -155,18 +162,18 @@ describe('Directors as a COOP', () => {
   it('displays the list of directors', () => {
     const directorListUI = vm.$el.querySelectorAll('.director-list-item')
 
-    expect(directorListUI[1].textContent).toContain('Peter')
-    expect(directorListUI[1].textContent).toContain('Griffin')
+    expect(directorListUI[0].textContent).toContain('Peter')
```

Comment:
Why did you change this unit test? Does it correctly verify that the director sorting is functioning?

Reply:
this was done before the new sorting ticket was merged, I will have to rebase and see if this is still valid